global/matlab/mixtureModel.m

```diff
@@ -0,0 +1,92 @@
+function mixtureModel(inFile,outFile,wbcmd,melocmd)
```

Comment:
This looks like it should be a bash script, I don't see any matrix math being done internally.  Perhaps the only reason it is a matlab script is because the place you currently want to call it from is matlab code?  Let's not use proprietary languages when they aren't the most obvious choice for the task.

Reply:
You are right, though I would argue that it also makes it more easy to debug 🐛. It is designed to be called from matlab scripts because in most use cases (It is supposed to be a more general utility, not just a helper function for this run_icasso) its inputs will be created by a matlab script. Also, in the future an option could be added to feed it a matrix directly rather than a file path, which bash couldn't do.

Reply:
If by "feed it a matrix" you mean "write that matrix to a file, run melodic, etc on it, and maybe read it back in later", I would still argue that the command sequence should be in a bash script, for more legibility, and potentially to be able to run the command sequence from a different script without having matlab/octave in the way.  If you wanted it to look more matlabby, you could write a secondary matlab wrapper that writes a matrix to a file and runs the bash script.  Unless a process actually uses matlab to do the math, I don't want its implementation to depend on matlab.  The less matlab we use in the pipelines, the easier they are to maintain and use (compiled matlab, MCR version, etc).